dvc/repo/experiments/remove.py

```diff
@@ -30,6 +30,7 @@ def remove(  # noqa: C901, PLR0912
     num: int = 1,
     queue: bool = False,
     git_remote: Optional[str] = None,
+    keep_selected: bool = False,  # keep the experiments instead of removing them
```

Comment:
let's name it just `keep` (to be aligned with the option name + it's more or less clear from the code what it does)

Reply:
Ok. done !